src/Microsoft.AspNet.OData/EnableQueryAttribute.cs

```diff
@@ -122,6 +155,35 @@ public override void OnActionExecuted(HttpActionExecutedContext actionExecutedCo
             }
         }
 
+        private static string ProcessActionArguments(HttpActionContext context)
```

Comment:
It seems that this method is only concerned with the generation of the expand query, for that reason I think the name `ProcessActionArguments` is too generic and the code would be easier to understand if it was renamed to something that better expresses the intent and scope of the method, e.g. `GenerateExpandQueryFromActionBody/Payload/Input` etc.

Reply:
Changed to `GenerateExpandQueryFromPayload`